### PR TITLE
switch to Julia v0.7

### LINKED ISSUE #12: Call @js fails in Julia 0.7
```
JSExpr.@js () -> begin
                  @var valcopy = JSON.parse(JSON.stringify(val))
              end
ERROR: LoadError: JSExpr: Unsupported `macrocall` expression, #= REPL[25]:2 =# @var valcopy = JSON.parse(JSON.stringify(val))
Stacktrace:
 [1] error at ./error.jl:33 [inlined]
 [2] macro expansion at /Users/vdayanan/.julia/packages/JSExpr/ySTC/src/JSExpr.jl:44 [inlined]
 [3] macro expansion at /Users/vdayanan/.julia/packages/MacroTools/ZXz1/src/macro.jl:18 [inlined]
 [4] jsexpr(::Expr) at /Users/vdayanan/.julia/packages/JSExpr/ySTC/src/JSExpr.jl:157
 [5] macro expansion at /Users/vdayanan/.julia/packages/JSExpr/ySTC/src/JSExpr.jl:44 [inlined]
 [6] macro expansion at /Users/vdayanan/.julia/packages/MacroTools/ZXz1/src/macro.jl:18 [inlined]
 [7] jsexpr(::Expr) at /Users/vdayanan/.julia/packages/JSExpr/ySTC/src/JSExpr.jl:157
 [8] _broadcast_getindex at ./broadcast.jl:587 [inlined]
 [9] getindex at ./broadcast.jl:520 [inlined]
 [10] copy at ./broadcast.jl:771 [inlined]
 [11] materialize at ./broadcast.jl:737 [inlined]
 [12] jsexpr_joined(::Array{Any,1}, ::String) at /Users/vdayanan/.julia/packages/JSExpr/ySTC/src/JSExpr.jl:26
 [13] block_expr(::Array{Any,1}, ::Any) at /Users/vdayanan/.julia/packages/JSExpr/ySTC/src/JSExpr.jl:33 (repeats 2 times)
 [14] jsexpr(::Expr) at /Users/vdayanan/.julia/packages/JSExpr/ySTC/src/JSExpr.jl:155
 [15] _broadcast_getindex at ./broadcast.jl:587 [inlined]
 [16] getindex at ./broadcast.jl:520 [inlined]
 [17] copy at ./broadcast.jl:771 [inlined]
 [18] materialize at ./broadcast.jl:737 [inlined]
 [19] jsexpr_joined(::Array{Any,1}, ::String) at /Users/vdayanan/.julia/packages/JSExpr/ySTC/src/JSExpr.jl:26
 [20] block_expr(::Array{Any,1}, ::Any) at /Users/vdayanan/.julia/packages/JSExpr/ySTC/src/JSExpr.jl:33 (repeats 2 times)
 [21] jsexpr(::Expr) at /Users/vdayanan/.julia/packages/JSExpr/ySTC/src/JSExpr.jl:155
 [22] func_expr(::Any, ::Any) at /Users/vdayanan/.julia/packages/JSExpr/ySTC/src/JSExpr.jl:87
 [23] macro expansion at /Users/vdayanan/.julia/packages/JSExpr/ySTC/src/JSExpr.jl:44 [inlined]
 [24] macro expansion at /Users/vdayanan/.julia/packages/MacroTools/ZXz1/src/macro.jl:18 [inlined]
 [25] jsexpr(::Expr) at /Users/vdayanan/.julia/packages/JSExpr/ySTC/src/JSExpr.jl:157
 [26] jsstring(::Any) at /Users/vdayanan/.julia/packages/JSExpr/ySTC/src/JSExpr.jl:29
 [27] @js(::LineNumberNode, ::Module, ::Any) at /Users/vdayanan/.julia/packages/JSExpr/ySTC/src/JSExpr.jl:9
in expression starting at REPL[25]:1 
```

### PR DESCRIPTION
**Pull request by rdeits**
This involves more subtle macro expansion changes, as I think `@__MODULE__()` is not exactly the same as `current_module()`, but it does appear to work so far...

Closes #12 (actually already fixed)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,31 +4,20 @@ os:
...
-#    - gfortran
-#before_script: # homebrew for mac
-#  - if [ $TRAVIS_OS_NAME = osx ]; then brew install gcc; fi
-
-## uncomment the following lines to override the default test script
-script:
-  - julia -e 'Pkg.clone("git://github.com/JuliaGizmos/WebIO.jl.git"); Pkg.clone(pwd()); Pkg.build("JSExpr"); Pkg.test("JSExpr"; coverage=true)'
 after_success:
   # push coverage results to Coveralls
-  - julia -e 'cd(Pkg.dir("JSExpr")); Pkg.add("Coverage"); using Coverage; Coveralls.submit(Coveralls.process_folder())'
```
Comment (tkoolen) [inline .travis.yml]: No need to `import JSExpr` here I don't think. It doesn't hurt though.

Comment (rdeits) [reply .travis.yml]: fixed

**Comment by rdeits** [comment]
Ok, all green!

**Comment by piever** [comment]
Can I merge this and tag a release?

**Comment by rdeits** [comment]
Yes please

**Pull request merged**